front-end-tools/src/components/InitComponent.tsx

```diff
@@ -0,0 +1,643 @@
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import Select from 'react-select';
+import { Alert, Button, Form, Row } from 'react-bootstrap';
+
+import { WalletConnection } from '@concordium/react-components';
+import {
+    ModuleReference,
+    TransactionKindString,
+    TransactionSummaryType,
+    displayTypeSchemaTemplate,
+    toBuffer,
+    ConcordiumGRPCClient,
+    getInitContractParameterSchema,
+} from '@concordium/web-sdk';
+
+import { TxHashLink } from './CCDScanLinks';
+import Box from './Box';
+import { initialize } from '../writing_to_blockchain';
+import { getObjectExample, getArrayExample } from '../utils';
+import { REFRESH_INTERVAL, INPUT_PARAMETER_TYPES_OPTIONS } from '../constants';
+
+interface ConnectionProps {
+    isTestnet: boolean;
+    account: string;
+    connection: WalletConnection;
+    client: ConcordiumGRPCClient | undefined;
+    contracts: string[];
+    embeddedModuleSchemaBase64: undefined | string;
+    moduleReferenceCalculated: undefined | string;
+    moduleReferenceDeployed: undefined | string;
+}
+
+export default function InitComponenet(props: ConnectionProps) {
```

Comment:
```suggestion
export default function InitComponent(props: ConnectionProps) {
```

Reply:
Also please remember to add a high-level comment to each of the components explaining its purpose.